src/System.IO.FileSystem/src/System/IO/Directory.cs

```diff
@@ -579,6 +579,11 @@ public static void Delete(String path, bool recursive)
             String fullPath = Path.GetFullPath(path);
             FileSystem.Current.RemoveDirectory(fullPath, recursive);
         }
+
+        public static string[] GetLogicalDrives()
```

Comment:
I'm still not understanding the hesitancy.  This introduces a competing mechanism for doing the PAL in this assembly.  There are already other members of FileSystem that are less-than-perfect (e.g. there's no "creation time" on Linux, yet we have Set/GetCreationTime).  Etc.  I don't know why we would do this differently.  The abstraction is still internal; if in the future we want to expose it, I expect at that time it would entail other changes anyway, so we should remain consistent with the abstraction for now and then revisit it if/when we ever expose it.